invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/records/search/SearchResultItemLayout.js

```diff
@@ -96,16 +96,24 @@ class SearchResultItemComponent extends Component {
           data-label={i18next.t("Files")}
           className="word-break-all"
         >
-          {humanReadableBytes(result.files.total_bytes)} | #{result.files.count}
+          {result.files.count} file{result.files.count !== 1 ? "s" : ""}:{" "}
```

Comment:
should we look into using some pluralisation lib ?

Reply:
Complicated because of translations.
@carlinmack we should mark all strings as translatable, see [doc](https://inveniordm.docs.cern.ch/develop/howtos/i18n/#singular-plural)

Reply:
It says: 

> This will generate the following two ids for translation:

Do I need to generate and add the English translation for this? Previously I've been told not to generate the translation files

Reply:
Nope, we only mark as translatable

Reply:
![image](https://github.com/user-attachments/assets/e1249c5d-b64e-4785-afe6-a8bbaf94036b)

hmm am I right that means that it would say `1 files` until 13.1?

Reply:
I guess that this is a screenshot after you have applied the translation changes. If that's the case, maybe yes you need to add the English translation. Something to have a look and solve ;)

Reply:
okay will do! who knows, maybe there's a translation fairy ✨

Reply:
![image](https://github.com/user-attachments/assets/3b737d7f-57a1-426a-8c74-806f8ad8f196)
appears we have some pluralisation errors in prod, will also fix

Reply:
there was another solution, using `(s)` for "optional" pluralisation as in 
```suggestion
          {result.files.count} file(s)
```
was this discussed as a possible solution?

Reply:
Why is this a possible solution? We have put effort in making translations with plurals.
What is the issue in using that?

`file(s)` is not translatable, and even if it is, it works in English only (and maybe other few languages that I am not aware of).

Reply:
related previous issue + PRs https://github.com/inveniosoftware/invenio-app-rdm/issues/2162

Reply:
From the conversation, and from trying out the different documented solutions, it seems that I18N plurals support needs more work.

@carlinmack (after the deployment), can you open an issue (or reopen and update any existing ones) with what you tried and existing examples that don't seem to be working?

Reply:
https://github.com/inveniosoftware/invenio-app-rdm/issues/2905

^ Will add more detail later